docs/update-prod-data.md

```diff
@@ -0,0 +1,82 @@
+# Steps to update the dev db with prod data
+
+
+## 1 - Take a Dump of Data from Production and Export to a File
+
+You'll first need to take a dump of the data from the production environment and export it to a file. This file will contain all of the data that is currently in the production database.
+
+You will first have to use **kubectl** to get the list of available pods `kubectl get pod`
+Locate the one with the prefix `lists-postgres- `
+
+The you'll have to access the shell of that pod with the following command:
+```bash
+kubectl exec -it lists-postgres-[pod_id]
+```
+
+_Replace [pod_id] with the reelevant id, eg `764c9b5879-5mf5f`_
```

Comment:
reelevant (install a spellchecker for VSC. It helps with code too).

Reply:
Spell checker is install I just must have missed it 😅

<img width="875" alt="Screenshot 2023-02-21 at 15 12 50" src="https://user-images.githubusercontent.com/1377253/220387502-8f52dc32-e041-4b46-807f-0a7fdd5a489d.png">